src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -379,6 +379,7 @@ Unit::Unit(bool isWorldObject) :
     _oldFactionId = 0;
     _isWalkingBeforeCharm = false;
     _instantCast = false;
+	_positionUpdateInfo = PositionUpdateInfo();
```

Comment:
@Shauren is this necessary?

Reply:
most likely no, I think if you remove everything the user defined constructor should be called
or if you remove it and choose for the inline initialize like I commented bellow it also should not be necessary

_should_

Reply:
https://arne-mertz.de/2015/08/new-c-features-default-initializers-for-member-variables/

2015, but yet an enjoyable reading

Reply:
https://www.fluentcpp.com/2018/06/15/should-structs-have-constructors-in-cpp/

this one too, fun fun 😄

Reply:
https://www.fluentcpp.com/2017/06/13/the-real-difference-between-struct-class/

![image](https://user-images.githubusercontent.com/4329612/72682487-bc4b9c80-3acd-11ea-9bc2-f3b5dda74552.png)

Reply:
(btw I posted these posts cause I genually look for them to know myself, I thought they might be handy for you or the lurkers, or both 😄)